styles/index.css

```diff
@@ -0,0 +1,78 @@
+form {
+  margin: 20px;
+  text-align: center;
+}
+
+input {
+  width: 400px;
+  height: 30px;
+  border-color: aqua;
+  font-size: 15px;
+}
+
+#weather,
+#reset {
+  width: 150px;
+  height: 30px;
+  border-color: lightblue;
+  margin: 15px;
+  font-size: 15px;
+  font-weight: bold;
+}
+
+#weather-app {
+  text-align: center;
+  margin-top: 30px;
+  margin-bottom: 10px;
+  margin-left: 500px;
+  background-color: lightcyan;
+  width: 550px;
+  height: 700px;
+  border-radius: 40px;
+  border-style: dotted;
+}
+
+h2,
+#current-weather {
+  font-size: 35px;
+}
+
+#temp,
+#farenheit {
+  font-weight: bolder;
+  font-size: 45px;
+}
+
+#city {
+  font-size: 30px;
+}
+
+#sky {
+  font-size: 40px;
+  text-align: center;
+}
+
+#emojis {
+  font-size: 35px;
+  text-align: center;
+  margin: 20px;
+}
+
+#increase,
```

Comment:
A little tip for CSS: If you find yourself using listing more than one id for the same set of CSS rules, you make want to use a class for them!